src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,241 @@
+package duke;
+
+import java.util.ArrayList;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.io.File;
+
+public class Duke {
+
+    public static ArrayList<Task> taskList = new ArrayList<>();
+
+    public static final Scanner SCANNER = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        greet();
+        loadFile();
+        while (true) {
+            String userInput = SCANNER.nextLine();
+            executeCommand(userInput);
+        }
+    }
+
+    private static void greet() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+
+        lineBreak();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        lineBreak();
+    }
+
+    public static void lineBreak() {
+        final String HORIZONTAL_LINE = "____________________________________________________________";
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void loadFile() {
+        File f = new File ("duke.txt");
+        Scanner s;
+        try {
+            s = new Scanner(f);
+        } catch (FileNotFoundException e) {
+            System.out.println("Data file does not exist");
+            lineBreak();
+            return;
+        }
+        System.out.println("Loading saved data");
+        lineBreak();
+        while (s.hasNext()) {
+            String line = s.nextLine();
+            int dividerPosition;
+            char taskType = line.charAt(1);
+            switch (taskType) {
+            case 'T':
+                taskList.add(new Todo(line.substring(7)));
```

Comment:
what does line.substring(7) stand for? maybe we should assign a variable for it?